src/coreComponents/python/modules/geosx_mesh_doctor/checks/elements_volumes.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+import logging
+from typing import List, Tuple
+
+from vtkmodules.vtkFiltersVerdict import (
+    vtkCellSizeFilter
+)
+from vtk.util.numpy_support import (
+    vtk_to_numpy,
+)
+
+
+from . import vtk_utils
+
+
+@dataclass(frozen=True)
+class Options:
+    min_volume: float
+
+
+@dataclass(frozen=True)
+class Result:
+    element_volumes: List[Tuple[int, float]]
+
+
+def __check(mesh, options: Options) -> Result:
+    f = vtkCellSizeFilter()
+
+    f.ComputeAreaOff()
+    f.ComputeLengthOff()
+    f.ComputeSumOff()
+    f.ComputeVertexCountOff()
```

Comment:
Needed?

Reply:
Since I think we do not use those data yet, I'd prefer not to compute them?

Reply:
Yes, you're right